### ISSUE TITLE
Wrong bin file been created

### ISSUE THREAD

**Issue by xlc** [cennznet/cli#11]
```
$ yarn global add @cennznet/cli --ignore-engines
yarn global v1.15.2
[1/4] 🔍  Resolving packages...
[2/4] 🚚  Fetching packages...
[3/4] 🔗  Linking dependencies...
[4/4] 🔨  Building fresh packages...

success Installed "@cennznet/cli@0.7.0" with binaries:
      - cennz-cli
      - cennz-cli.js
✨  Done in 4.63s.
```

```
$ cennz-cli
internal/modules/cjs/loader.js:670
    throw err;
    ^

Error: Cannot find module '/Users/bryanchen/.config/yarn/global/node_modules/.bin/run.js'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:668:15)
    at Function.Module._load (internal/modules/cjs/loader.js:591:27)
    at Function.Module.runMain (internal/modules/cjs/loader.js:877:12)
    at internal/main/run_main_module.js:21:11
```

```
$ cennz-cli.js
Official CLI tool to interact with CENNZnet nodes.

VERSION
  @cennznet/cli/0.7.0 darwin-x64 node-v11.14.0

USAGE
  $ cennz-cli [COMMAND]

COMMANDS
  api     Send transactions
  help    display help for cennz-cli
  repl    Start a repl to interact with a node
  script  manage scripts
  wallet  manage your wallet
```

Should only have cennz-cli one

**Comment by ianhe8x** [comment]
Very interesting. i use `npm -g` to install it, and npm creates a soft link from cennz-cli to /Users/xxx/.nvm/versions/node/v10.13.0/lib/node_modules/@cennznet/cli/bin/cennz-cli
then i tried `yarn global`, it links `cennz-cli` to a different place `/.config/yarn/global/node_modules/.bin/cennz-cli`.

Will find a workaround for this case.

**Comment by ianhe8x** [comment]
fixed in v0.7.1
